core/app/models/concerns/spree/default_price.rb

```diff
@@ -16,9 +16,15 @@ def self.default_price_attributes
 
     # Returns `#prices` prioritized for being considered as default price
     #
-    # @return [ActiveRecord::Relation<Spree::Price>]
+    # @return [Array<Spree::Price>]
     def currently_valid_prices
-      prices.currently_valid
+      prices.sort_by do |price|
```

Comment:
Any chance we can reorganize logic so that it's not duplicated with https://github.com/nebulab/solidus/blob/60f446c4343ad3929e249af0fb4ecf948eb3fbad/core/app/models/concerns/spree/default_price.rb#L68?

Reply:
`prioritized_default` is deleted in the following commit ;)

Reply:
Oh... 🙂